pkg/reconciliation/reconcile_racks.go

```diff
@@ -617,11 +618,13 @@ func (rc *ReconciliationContext) CheckPodsReady(endpointData httphelper.CassMeta
 		return result.RequeueSoon(5)
 	}
 
-	needsMoreNodes, err := rc.startAllNodes(endpointData)
+	// step 4 - make sure all nodes are ready
```

Comment:
Now the step 3 comment is wrong as it used to be this step.

Reply:
OK will fix comment for step 3